LEDControl/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## [1.0.0] - 2022-16-12
+### Added
+- New RDK Service LEDControl to manage LEDs in the device front panel
+
+### Change
```

Comment:
Pls remove the ##Change label and comments below it. This was added for previous plugins that were present before we added the CHANGELOG. 
Also pls add the generic instructions at the top of the CHANGELOG similar to others - https://github.com/rdkcentral/rdkservices/blob/sprint/22Q4/Bluetooth/CHANGELOG.md